README.md

```diff
@@ -85,3 +85,11 @@ You can perform a more thorough test by running the test script from the reposit
 
     %> ./run_tests.sh 
 
+
+Archive
+=======
+
+We archive our old research code and experiments in the following repository:
```

Comment:
"Some of our old research code and experiments are archived in the following repository:"